examples/NavigationPlayground/js/Drawer.js

```diff
@@ -39,11 +39,10 @@ const InboxScreen = ({ navigation }) => (
 InboxScreen.navigationOptions = {
   drawer: {
     label: 'Inbox',
-    icon: ({ tintColor }) => (
```

Comment:
Yeah, but that was to demonstrate it can be a non-function as well, but will revert.